index.js

```diff
@@ -45,14 +46,13 @@ var new_client = function(sdk_key, config) {
   );
   config.feature_store = config.feature_store || InMemoryFeatureStore();
 
-  queue = [];
-
   if (!sdk_key && !config.offline) {
     throw new Error("You must configure the client with an SDK key");
   }
 
   if (!config.use_ldd && !config.offline) {
-    requestor = Requestor(sdk_key, config);
+    var requestor = Requestor(sdk_key, config);
```

Comment:
minor: we should move these bindings up to the top of the scope to avoid a similar issue in the future.